src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -64,4 +81,60 @@ public Response getDatasetByIdentifier(@Auth AuthUser authUser, @PathParam("iden
             return createExceptionResponse(e);
         }
     }
+
+    @POST
+    @Consumes("application/json")
+    @Produces("application/json")
+    @Path("/dar/draft")
+    @RolesAllowed({ADMIN})
+    public Response createDraftDataAccessRequest(
+        @Auth AuthUser authUser,
+        @Context UriInfo info,
+        @QueryParam("identifiers") String identifiers,
+        @QueryParam("projectTitle") String projectTitle) {
+      try {
+        // Ensure that the user is a registered DUOS and SAM user
+        User user = findOrCreateUser(authUser);
+        if (Objects.isNull(identifiers) || identifiers.isBlank()) {
+          throw new BadRequestException("No dataset identifiers were provided");
+        } else {
+          List<Integer> datasetIds = Arrays.asList(identifiers.split(","))
+              .stream()
+              .filter(identifier -> !identifier.isBlank())
+              .map(identifier -> this.datasetService.findDatasetByIdentifier(identifier).getDataSetId())
+              .collect(Collectors.toList());
+        if (datasetIds.isEmpty()) {
+            throw new BadRequestException("No datasets associated with identifiers: " + identifiers);
+        }
+        DataAccessRequest newDar = new DataAccessRequest();
+        DataAccessRequestData data = new DataAccessRequestData();
+        String referenceId = UUID.randomUUID().toString();
+        newDar.setReferenceId(referenceId);
+        data.setReferenceId(referenceId);
+        data.setProjectTitle(projectTitle);
+        newDar.setData(data);
+        newDar.setDatasetIds(datasetIds);
+        DataAccessRequest result = this.darService.insertDraftDataAccessRequest(user, newDar);
+        URI uri = info.getRequestUriBuilder().path("/" + result.getReferenceId()).build();
+        return Response.created(uri).entity(result.convertToSimplifiedDar()).build();
+        }
+      } catch (Exception e) {
+          return createExceptionResponse(e);
+      }
+    }
+
+    // should this be private? otherwise can't test
```

Comment:
could make this a service method in TDRService ?

Reply:
Agreed, this should probably be a service method. I think it's better handled in `UserService` - might need some more work there to make that happen. Sam utilities are in their own service, not a DAO, so making `UserService` dependent on `SamService` could introduce a circular dependency. Have not checked, but that is a potential issue. On the other hand, if we moved this to `SamService`, we could add `userDAO` there and avoid any dependency problems.

Reply:
Another option here is to introduce another layer between `SamService` and a new `SamDAO`. In effect, we would:

1. rename `SamService` to `SamDAO`
2. create a new `SamService` that has a `SamDAO`
3. re-implement the same methods in the new version of the service with passthroughs to the dao

That way, any service could then pull in a `SamDAO` and not have to worry about circular dependencies. I wish I had thought of this in the original implementation - would have made this kind of work a lot easier.

Reply:
See https://github.com/DataBiosphere/consent/pull/1745 for a potential solution

Reply:
Merged changes from [1745](https://github.com/DataBiosphere/consent/pull/1745) and implemented a new UserService method to handle looking up and creating a user with the newly available SamDAO.